### PR TITLE
[TESTS] Check warnings

### PR DESCRIPTION
**Pull request by rousseldenis**

### CODE REVIEW

**Comment by rousseldenis** [comment]
@sbidoul @pedrobaeza @OCA/core-maintainers This is a proof of concept that shows how to trigger a failing check on PR's main page in order to warn people about current errors (showed also in tests) and warnings (the main goal of this).

I found that a lot of repositories have bunches of warnings that are never addressed, but should.

The idea is to warn earlier (before merge) people about those warnings. As github has no intermediate (just fail/success) status for checks we cannot do that in normal checks and has to be on separate one.

The idea is to collect logs across matrix steps and run checkog-odoo. The difficult part was to generate unique log file id (if someone has better idea).

Comments are welcome !

**Comment by hbrunn** [comment]
yes please!

**Comment by pedrobaeza** [comment]
Well, it sounds good to have it. Any drawback with the storage? Do we have a limit?

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -81,10 +87,53 @@ jobs:
...
+            echo 'FILELIST<<EOF'
+            find . -name '*.log' -print 
+            echo EOF
+          } >> "$GITHUB_ENV"
+      - name: Check logs
+        run: |
+          pip install checklog-odoo
+          echo $FILELIST
+          {
+          for x in $FILELIST; do
```
Comment (sbidoul) [inline .github/workflows/test.yml:136]:
```suggestion
          for x in $(ls *.log); do
```

You can probably do this in one step like this.

**Comment by sbidoul** [comment]
@rousseldenis that's nice. Related: https://github.com/OCA/oca-addons-repo-template/issues/120

We could also enable this with an environment variable in oca-ci: https://github.com/OCA/oca-ci/issues/10

**Pull request closed without merging**